README.md

```diff
@@ -73,7 +68,7 @@ rsx! {
 You can easily declare your own custom widgets:
 ```rust
 #[widget]
-pub fn MyCustomWidget(children: Children) {
+pub fn MyCustomWidget(props: MyCustomWidgetProps) {
     rsx! {
```

Comment:
I think the other way works but it uses the "hidden children," so maybe we get the children directly like this:
```suggestion
pub fn MyCustomWidget(props: MyCustomWidgetProps) {
    let children = props.get_children();
    rsx! {
```